docs/usage.rst

```diff
@@ -419,26 +562,175 @@ To run Twisted in the main thread, just start Dispersy in your main thread
 
 .. code-block:: python
 
-    import signal
     from twisted.internet import reactor
 
     def main():
         reactor.exitCode = 0
         reactor.run()
 
-        dispersy = Dispersy(StandaloneEndpoint(port, '0.0.0.0'), unicode(data_dir), u'dispersy.db')
+        dispersy = Dispersy(StandaloneEndpoint(port, '0.0.0.0'), unicode(<data_dir>), u'dispersy.db')
         dispersy.statistics.enable_debug_statistics(True)
         dispersy.start(autoload_discovery=True)
 
         my_member = self.get_new_member()
-        master_memeber = self.get_member(public_key=master_key)
+        master_memeber = self.get_member(public_key=<master_key>)
 
-        community = Community.init_community(self, master_member, my_member)
+        community = <Community>.init_community(self, master_member, my_member)
 
         exit(reactor.exitCode)
 
     if __name__ == "__main__":
         main()
 
+The variables between <> have to be replaced with values/objects belonging to your own project.
+
 Run Twisted in a separate thread
 --------------------------------
+
+To run Dispersy in a separate thread you have to run twisted in a separate thread. This can be done by overriding the
+start and stop functions of twisted as can be seen below. This code is best placed in a separate file. The import to
+the start_reactor() function should be the first import to the twisted library in your project otherwise the twisted
+library is already initialised and will be running on the same thread.
+
+.. code-block:: python
+
+    from twisted.internet import reactor
+    from twisted.python import log
+
+    _twisted_thread = None
+
+
+    def start_reactor():
```

Comment:
I don't want to have this on the documentation, it will bitrot here.

Please link to the original source instead.